### PR TITLE
Possibility to update video streaming capabilities during ignition cycle

### PR DESCRIPTION
**Pull request by dboltovskyi**
The main purpose of this proposal is to make it possible to update video streaming capabilities during the ignition cycle.

### CODE REVIEW

**Comment by dboltovskyi** [comment]
@theresalech Please notice this proposal is ready for review.

File: proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Mobile applications should be able to update the streaming content window to the new video streaming capabilities received in `OnSystemCapabilityUpdated`.
+
+## Potential downsides
+
+N/A
+
+## Impact on existing code
+
+There should be a minimal impact on existing code.
```
Comment (theresalech) [inline proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: As Core, iOS, and Java Suite are all included as impacted platforms, please provide more information about the impact to these.  For example, will this be a minor version change in each repository?

File: proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+![Sequence diagram](/assets/proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle/sequence_diagram.png)
+
+### Mobile libraries changes (Android and iOS)
+
+Mobile applications should be able to update the streaming content window to the new video streaming capabilities received in `OnSystemCapabilityUpdated`.
+
+## Potential downsides
+
+N/A
```
Comment (theresalech) [inline proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: It's recommended to complete each section of the proposal template.  If you are unable to provide potential downsides, can you please update this to "The author was unable to identify any potential downsides."?

File: proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+## Impact on existing code
+
+There should be a minimal impact on existing code.
+
+## Alternatives considered
+
+N/A
```
Comment (theresalech) [inline proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: It's recommended to complete each section of the proposal template.  If you are unable to provide alternatives considered, can you please update this to "The author was unable to identify any alternative solutions."?

File: proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md (hunk 1 above)
Comment (dboltovskyi) [reply proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: @theresalech Please find update in 7186d5e8d8ef547f2ce22d5d7b55f2dfba47628e

File: proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md (hunk 3 above)
Comment (dboltovskyi) [reply proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: @theresalech Please find update in 7186d5e8d8ef547f2ce22d5d7b55f2dfba47628e

File: proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md (hunk 2 above)
Comment (dboltovskyi) [reply proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: @theresalech Please find update in 7186d5e8d8ef547f2ce22d5d7b55f2dfba47628e

File: proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
+# Possibility to update video streaming capabilities during ignition cycle
+
+* Proposal: [SDL-NNNN](NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md)
+* Author: [Dmytro Boltovskyi](https://github.com/dboltovskyi)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core / iOS / Java Suite]
+
+## Introduction
+
+The main purpose of this proposal is make possible to update video streaming capabilities during the ignition cycle.
```
[the remark this answers is not in the document]
Comment (theresalech) [reply proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: Please update the PR description to:
> The main purpose of this proposal is to make it possible to update video streaming capabilities during the ignition cycle.

[the remark this answers is not in the document]
Comment (dboltovskyi) [reply proposals/NNNN-Update-video-streaming-capabilities-during-ignition-cycle.md]: @theresalech Thank you for noticing this. Description is updated.

**Comment by dboltovskyi** [comment]
@theresalech Thank you for the review and comments. Description is updated.

**Pull request merged**